cypress/e2e/specificDrinks.cy.js

```diff
@@ -38,16 +38,13 @@ describe('Specific spec', () => {
         // cy.get('.ingredientsTitle').contains('Amaretto Stone Sour Alternative') 
     })
 
-    it.only('should return an error message if a network request fails', () => {
-        cy.visit('http://localhost:3000/drinks/16100')
-        cy.intercept('GET', 'https://www.thecocktaildb.com/api/json/v1/1/lookup.php?i=16100', {
-        statusCode: 404,
-        body: {
-          error: 'Cannot GET /lookup.php?i=16100'
-        }
-      })
-      .get('Error').should('have.value', 'Not Found')
-  })
-
+    it('should return an error message if a network request fails to fetch specialty drinks', () => {
```

Comment:
This one gave us some trouble but we all pitched in a different piece of the solution which was nice to see. Great teamwork